backend/climateconnect_api/views/user_views.py

```diff
@@ -95,15 +95,11 @@ def get(self, request):
 
 
 class MemberProfilesView(ListAPIView):
-    permission_classes = (IsAuthenticated,)
+    permission_classes = [AllowAny]
     serializer_class = UserProfileSerializer
     pagination_class = PageNumberPagination
     filter_backends = [SearchFilter]
     search_fields = ['url_slug']
 
     def get_queryset(self):
-        if not UserProfile.objects.filter(user=self.request.user).exists() or \
-                not self.request.user.user_profile.is_profile_verified:
-            raise PermissionDenied(detail="You do not have permission to access this page.")
-
```

Comment:
I think we should ask memebers at sign in if they allow to show their profile publically or just to other climate connect members and organizations!